### PR TITLE
Add coverage badge into readme 

### PR DESCRIPTION
**Pull request by akanksha-maker-ucsd**
add jsDocs deployment link

### CODE REVIEW

**Comment by alexzhang1618** [comment]
Changes look good! One question: instead of pushing JSDocs to a different repo, can't we just merge the gh-pages branch into main so jsDocs are tied into the main site?

**Comment by akanksha-maker-ucsd** [comment]
> Changes look good! One question: instead of pushing JSDocs to a different repo, can't we just merge the gh-pages branch into main so jsDocs are tied into the main site?
 
we tried this, but merging onto main is blocked due to our review process.

File: README.md [hunk 1]
```diff
@@ -18,9 +20,10 @@ Personal note taking app that enables users to share their notes with other user
 
 Video and tutorial on how our project works.
 
-## Documentation
+## Documentation
```
Comment (alexzhang1618) [inline README.md:23]: Can we remove these changes here since we're deploying documentation through the main page? Otherwise looks good to me!

**Review by alexzhang1618 — APPROVED** [review]
Note: Merged and rebased with main (might be why there are so many commits in the history) and added skip_git_check to the workflow to prevent `no git repository detected` errors by the HTML validator. 
Looks good to me!

**Pull request merged**